packages/brand-positioning-block/src/components/Item.tsx

```diff
@@ -0,0 +1,40 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { joinClassNames } from '@frontify/guideline-blocks-shared';
+import { useDraggable } from '@dnd-kit/core';
+import { ItemProps } from '../types';
+import { Toolbar } from './Toolbar';
+
+export const Item = ({ src, x, y, id }: ItemProps) => {
+    const editing = true;
+    const { attributes, isDragging, listeners, setNodeRef, transform } = useDraggable({
+        id,
+    });
+    const transformStyle = {
+        transform: transform ? `translate(${transform.x}px, ${transform.y}px)` : '',
+        zIndex: isDragging ? 2 : 1,
+    };
+
+    return (
+        <div
+            style={{
+                left: `${x}%`,
+                top: `${y}%`,
+                ...transformStyle,
+            }}
+            ref={setNodeRef}
+            className="tw-h-[50px] tw-w-[50px] tw-bg-white tw-group tw-rounded tw-border tw-border-line tw-p-1 tw-absolute hover:tw-outline-offset-[1px] hover:tw-outline hover:tw-outline-[1px] hover:tw-outline-box-selected-inverse"
+        >
+            <div
+                className={joinClassNames([
+                    'tw-absolute tw-z-20 tw-bottom-[calc(100%-4px)] tw-right-[-3px] tw-w-full',
```

Comment:
Why not just `top: 4px` there?

Reply:
Please use multiple of 4 only for `px` value